services/workflows-service/src/providers/file/file.service.ts

```diff
@@ -266,22 +272,26 @@ export class FileService {
     const remoteFileNamePrefix = fileDetails.id || getDocumentId(fileDetails, false);
 
     let tmpLocalFilePath;
+
     if (shouldDownloadFromSource) {
       const tmpFile = tmp.fileSync();
+
       tmpLocalFilePath = await sourceServiceProvider.download(sourceRemoteFileConfig, tmpFile.name);
     }
 
-    const file = await fromFile(tmpLocalFilePath ?? fileDetails.uri);
-
+    const fileType = await getFileMetadata({
+      file: tmpLocalFilePath || fileDetails.uri,
+      fileName: fileDetails.fileName || fileDetails.uri || '',
```

Comment:
You can have uri in file name column, try to extract the file name:

```
function extractFileNameFromUri(uri) {
  // Use the path module to extract the file name
  return path.basename(uri);
}
```

`extractFileNameFromUri(fileDetails.uri)`